amgut/db/patches/0041.sql

```diff
@@ -56,24 +105,28 @@ CREATE TABLE pm.processing_robot (
     CONSTRAINT uq_processing_robot_name UNIQUE ( name )
  );
 
-CREATE TABLE pm.protocol (
-    protocol_id          bigserial  NOT NULL,
-    name                 varchar  NOT NULL,
-    CONSTRAINT pk_protocol PRIMARY KEY ( protocol_id ),
-    CONSTRAINT uq_protocol_name UNIQUE ( name )
- );
+ CREATE TABLE pm.sequencer (
+     sequencer_id         bigserial  NOT NULL,
+     platform             varchar  NOT NULL,
+     instrument_model     varchar  NOT NULL,
```

Comment:
should `platform` and `instrument_model` be controlled? `enum` as the growth is small?